components/org.wso2.carbon.identity.api.server.organization.user.sharing.management/org.wso2.carbon.identity.api.server.organization.user.sharing.management.common/src/main/java/org/wso2/carbon/identity/api/server/organization/user/sharing/management/common/UserSharingMgtConstants.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.organization.user.sharing.management.common;
+
+/**
+ * Holds the constants which the shared user invitation management API component is using.
+ */
+public class UserSharingMgtConstants {
+
+    public static final String ERROR_PREFIX = "OUI-";
```

Comment:
```suggestion
    public static final String ERROR_PREFIX = "USM-";
```
User Sharing Management

Reply:
Addressed in commit: [cabd548](https://github.com/wso2/identity-api-server/pull/784/commits/cabd548d446c83c21e70c8e955aabd33265c32a0)